src/dcmp/dcmp.c

```diff
@@ -948,10 +948,14 @@ static int dcmp_strmap_compare(mfu_flist src_list,
          * we can do this comparison in parallel more effectively.  For
          * now copy these files to the list of files we need to compare. */
 
-        /* make a copy of the src and dest files where the data needs
-         * to be compared and store in src & dest compare lists */
-        mfu_flist_file_copy(src_list, src_index, src_compare_list);
-        mfu_flist_file_copy(dst_list, dst_index, dst_compare_list);
+        /* Make a copy of the src and dest files where the data needs
+         * to be compared and store in src & dest compare lists. We
+         * dont need to bother copying into compare list if lite
+         * option is on */
+        if (options.lite) {
```

Comment:
should this be?
```
if (! options.lite)
```

Reply:
@adammoody yep. thank you. I just updated it.